opm/simulators/wells/StandardWellPrimaryVariables.cpp

```diff
@@ -50,9 +53,15 @@ namespace {
 //! \brief Relaxation factor considering only one fraction value.
 template<class Scalar>
 Scalar relaxationFactorFraction(const Scalar old_value,
-                                const Scalar dx)
+                                const Scalar dx,
+                                const std::string& well_name,
+                                const std::string& pv_name,
+                                Opm::DeferredLogger& deffered_logger)
```

Comment:
one f

Reply:
thanks. will address it when jenkins is done.

Reply:
> thanks. will address it when jenkins is done.

Cool.  Would you mind adding doxygen-style descriptions of the parameters too, please?  That might help future maintainers get a handle on the meainings of (e.g.) `pv_name` here.